src/components/KeyblockDetailsPanel.vue

```diff
@@ -28,7 +28,32 @@
             Height
           </th>
           <td class="keyblock-details-panel__data">
-            {{ keyblockDetails.height }}
+            <div class="keyblock-details-panel__controls">
+              <app-link :to="`/keyblocks/${keyblockDetails.height - 1}`">
+                <button
+                  :class="[
+                    'keyblock-details-panel__button',
+                    'keyblock-details-panel__button--prev',
+                  ]">
+                  <app-icon
+                    :size="22"
+                    name="caret-left"/>
+                </button>
+              </app-link>
+              {{ keyblockDetails.height }}
+              <app-link :to="`/keyblocks/${keyblockDetails.height + 1}`">
+                <button
+                  :class="[
+                    'keyblock-details-panel__button',
+                    'keyblock-details-panel__button--next',
+                  ]"
+                  :disabled="!isNextKeyblockMined">
```

Comment:
the button is still clickable, even if it's disabled :/

Reply:
Good catch. You are right, haven't spotted this. Fixed